index.html

```diff
@@ -156,6 +156,20 @@ <h1 class="mt-3 mb-3">Random Image Picker</h1>
                         <img id="random-image" class="img-thumbnail random-image" src="" alt="Randomly picked image.">
                     </div>
                 </div>
+                <div class="col-lg-4 mb-4 text-center">
+                    <div class="row ml-5">
+                        <h2 class="mt-3 mb-4">History:</h2>
```

Comment:
I would propose to have this title styled a bit better. I know The application doesn't have the nicest UI, but still... 😉

Reply:
Any suggestions, I'm not that good with design XD